apps/container/src/views/Performance/PerformanceView.vue

```diff
@@ -1,22 +1,312 @@
 <template>
-  <v-row>
-    <v-col cols="12">
-      <h1>Performance</h1>
-      <h2>Your info in Vue: {{ user?.email }}</h2>
-      <HelloWorld />
-    </v-col>
-  </v-row>
+  <CenteredLoading
+    class="mt-10"
+    v-if="
+      isPendingCrHistory || isPendingCpHistory || isPendingCrDistributionData
+    "
+  />
+  <v-layout class="flex-column align-center justify-center" v-else>
+    <v-row align="stretch" no-gutters class="w-100">
+      <v-col
+        v-for="card in cards"
+        :key="card.title"
+        cols="12"
+        sm="3"
+        class="mb-2 mb-sm-0"
+      >
+        <PerformanceCard
+          :title="card.title"
+          :subTitle="card.subtitle"
+          :description="card.content"
+          :color="card.color"
+          :icon="card.icon"
+          :tooltip="card?.tooltip"
+        >
+        </PerformanceCard>
+      </v-col>
+    </v-row>
+    <PaperCard class="w-100 mt-4">
+      <Chart :options="crHistoryOptions" />
+    </PaperCard>
+    <PaperCard class="w-100 mt-4">
+      <v-select
+          :items="cpHistoryData"
+          :item-title="(course) => course.curso"
+          :item-value="(course) => course"
+          v-model="currentCpCourse"
+          variant="outlined"
+          class="course-select"
+        />
+      <Chart :options="cpHistoryOptions" />
+    </PaperCard>
+    <PaperCard class="w-100 mt-4">
+      <Chart :options="crDistributionOptions" />
+    </PaperCard>
+  </v-layout>
 </template>
+
 <script setup lang="ts">
-import { defineFederatedReactComponent } from '@/utils/moduleFederation';
-import useAuth from '@/store/useAuth'
+import { ref, computed } from 'vue';
+import { Chart } from 'highcharts-vue';
+import { useQuery } from '@tanstack/vue-query';
+import PerformanceCard from '@/components/PerformanceCard.vue';
+import { Performance, type CourseInformation } from 'services';
+import { PaperCard } from '@/components/PaperCard';
+import { theme } from '@/bootstrap';
+import { CenteredLoading } from '@/components/CenteredLoading';
+import { formatSeason } from 'utils';
+
+const areaGraphOptions = {
+  accessibility: {
+    enabled: true,
+  },
+  chart: {
+    type: 'area',
+    style: { fontFamily: 'Roboto, sans-serif' },
+  },
+  plotOptions: {
+    area: {
+      fillOpacity: 0.45,
+      lineWidth: 2,
+      marker: {
+        radius: 4,
+      },
+    },
+  },
+  colors: [theme.colors?.primary],
+  tooltip: {
+    borderRadius: 10,
+    padding: 12,
+  },
+};
 
-const { user } = useAuth();
+const { data: crHistoryData, isPending: isPendingCrHistory } = useQuery({
+  queryKey: ['crHistory'],
```

Comment:
o padrão de query keys atual é baseado na rota e nos parametros

```suggestion
  queryKey: ['users', 'me', 'grades'],
```